generators/form/index.js

```diff
@@ -25,6 +25,11 @@ module.exports = class extends Generator {
         type: 'input',
         name: 'formNumber',
         message: "What's your form number? Examples: '22-0993' or '21P-530'",
+        // Replace spaces and underscores with dashes
+        // Convert to ALL CAPS
```

Comment:
No problem. I like to be overly descriptive to try to prevent misunderstandings as much as possible. When I see 'uppercase' I wonder if that means converting to ALL uppercase or titlecase?